prow/config/templates/plugins-ConfigMap.yaml

```diff
@@ -18,19 +18,25 @@ metadata:
   name: plugins
 data:
   plugins.yaml: |
+    triggers:
+    - repos:
+      - {{ .Values.github.organisation }}
+      only_org_members: true
```

Comment:
This is a key-value pair as part of the repos list?

Reply:
No, it exists alongside the repos list inside Trigger struct.
and "triggers" is list of Trigger struct.
https://sourcegraph.com/github.com/kubernetes/test-infra/-/blob/prow/plugins/config.go#L380:1

Reply:
Ugh reading YAML can be a pain in the butt sometimes. I see it now, though.